tools/src/wyvern/tools/tests/OOPSLA16.java

```diff
@@ -39,8 +40,9 @@
     }
 
 	@Test
+	@Category(CurrentlyBroken.class)
```

Comment:
The problem with these seem to be that objects representing resource modules are not resources, and thus fixing them will involve changes to module imports and/or module-to-object translation. They should be fixed as part of a PR dealing with resourceness of modules, instead of this one.